packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1613,11 +1539,11 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       return;
     }
 
-    const input = event.target as HTMLInputElement;
+    // const input = event.target as HTMLInputElement;
 
-    if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
+    if (event.code === 'ArrowUp' || event.code === 'ArrowDown') {
       this.handleArrowSelection(event);
-      if (input.value.length > 0) {
+      if (this.input?.value?.length > 0) {
```

Comment:
do we need to option chain on input here or just `this.input.value?` 
We have a direct access call to this.input.value below and no intervening guard clauses that would ensure that this.input.value is defined.
Optional chaining is great, but we should try only to use it when needed as there is a cost for it in the transpiled code